apps/consulting/posts/openai-pseudocode.md

```diff
@@ -0,0 +1,274 @@
+---
+title: 'Build your own code copilot with LLMs'
+published: March 27, 2023
+author: chris-ostrouchov
+description: 'As an experiment we developed a python package pseudocode which much like its name implies allows developers to annotate their code with pseudocode and produce testable code generated by LLMs without the user writing any code.'
+category: ["artificial-intelligence"]
```

Comment:
This needs to match the `name` property of the category object, and should not have enclosing double quotes.

```suggestion
category: [Artificial Intelligence]
```